test/motranslator/TokenParserTrans.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * (c) 2021 phpMyAdmin contributors
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace PhpMyAdmin\Tests\Twig\Extensions\motranslator;
```

Comment:
Why `motranslator` and not `MoTranslator`?

Reply:
I fixed that, no idea why it was not camel cased